Tests/SwiftDocCTests/Checker/Checkers/NonInclusiveLanguageCheckerTests.swift

```diff
@@ -31,6 +31,40 @@ class NonInclusiveLanguageCheckerTests: XCTestCase {
         XCTAssertEqual(range.upperBound.column, 16)
     }
 
+    func testMatchTermWithSpaces() throws {
+        let source = """
+# A White  listed title
+# A Black    listed title
+# A White listed title
+"""
```

Comment:
```suggestion
        # A White  listed title
        # A Black    listed title
        # A White listed title
        """
```

Would recommend aligning the end of the string here to the start column-wise to align with the style of the rest of the repo.

Reply:
sounds good. done 👍🏻